instagram/src/components/Authentication/Authentication.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import LoginPage from '../LoginPage/LoginPage'
+import { getPositivePatterns } from '../../../../../../AppData/Local/Microsoft/TypeScript/3.1/node_modules/fast-glob/out/managers/tasks';
+const Authenticate = App =>
+    class auth extends React.Component {
+        constructor() {
+            super()
+            this.state = {
+                user: '',
+                pass: '',
+                loginStatus: false
+            }
+        }
+        userHandler = event => {
```

Comment:
you can refactor userHandler and passHandler into one method by adding a name property of 'user' and 'pass' to each of your inputs, then create a handleChange method like this: 
```
handleChange = event => {
    this.setState({ [event.target.name]: event.target.value })
}
```